play-json/src/test/scala/play/api/libs/json/JsonValidSpec.scala

```diff
@@ -216,6 +216,49 @@ class JsonValidSpec extends Specification {
       JsString("alphabeta").validate[String] must equalTo(JsSuccess("alphabeta"))
     }
 
+    "validate reads on the root path" in {
```

Comment:
Done -- I've also changed some other tests in the same file that applied the same pattern.